build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation('org.apache.commons:commons-lang3:3.13.0')
 
     testImplementation('org.junit.jupiter:junit-jupiter:5.10.0')
+    testRuntimeOnly('org.junit.platform:junit-platform-launcher')
```

Comment:
Starting from Gradle 9.0 test framework dependencies must be manually declared on the test runtime classpath: https://docs.gradle.org/8.3/userguide/upgrading_version_8.html#test_framework_implementation_dependencies